src/cpp/cpp_typecheck.cpp

```diff
@@ -196,7 +196,7 @@ void cpp_typecheckt::static_and_dynamic_initialization()
 
   dynamic_initializations.clear();
 
-  // block_sini.move_to_operands(block_dini);
+  // block_sini.add_to_operands(std::move(block_dini));
```

Comment:
Should this line just be deleted, given that it's commented out?

Reply:
Possibly so, but with the C++ front-end there is just a lot of cleanup to be done and there may be genuine value in some of the code currently commented out. So I'd rather leave this to be done in other PRs.

Reply:
I'd prefer using an ifdef 0 plus a comment that explains why the code is commented out/worth keeping.

Reply:
It seems neither `block_sini` nor `block_dini` exist in anywhere in the code base, I have thus completely removed the line as it is impossible to make sense of it.